frontend-react/src/components/Table/ColumnData.tsx

```diff
@@ -74,16 +75,15 @@ export const ColumnData = ({
         // Render column value as NavLink
         const feature = columnConfig?.feature as LinkableColumn;
         return tableData(
-            <NavLink
-                className="usa-link"
-                to={`${feature.linkBasePath || ""}${
+            <USLink
```

Comment:
Great time to address `Link` and `NavLink`: from what I can find/read, the _only_ time we should utilize `NavLink` is for navigation headers such as our nav bar, a side nav display, or anything requiring the ability to display an `active` set of styles.

Reply:
Yep, NavLink is just a souped-up Link: https://github.com/remix-run/react-router/blob/main/packages/react-router-dom/index.tsx#L485-L543

It's surprisingly a lot of code to detect/implement current path activeness 😆